misc-tools/dj_make_chroot.in

```diff
@@ -279,6 +277,8 @@ echo "Running debootstrap to install base system, this may take a while..."
 eval PATH="$PATH:/bin:/sbin:/usr/sbin" $BOOTSTRAP_COMMAND $INCLUDEOPT $EXCLUDEOPT \
     --variant=minbase --arch "$ARCH" "$RELEASE" "$CHROOTDIR" "$DEBMIRROR"
 
+([ -n "$WORKDIR" ] && rm -rf "$WORKDIR") || /bin/true
```

Comment:
I think it's clearer to write this as:
```bash
if [ -n "$WORKDIR" ]; then
    rm -rf "$WORKDIR"
fi
```
Nitpick: maybe also better to test with `-d`.